samples/addons/kiali.yaml

```diff
@@ -326,20 +360,69 @@ subjects:
   namespace: istio-system
 ...
 ---
+# Source: kiali-server/templates/role-controlplane.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: kiali-controlplane
+  namespace: istio-system
+  labels:
+    helm.sh/chart: kiali-server-1.29.0
+    app: kiali
+    app.kubernetes.io/name: kiali
+    app.kubernetes.io/instance: kiali-server
+    version: "v1.29.0"
+    app.kubernetes.io/version: "v1.29.0"
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/part-of: "kiali"
+rules:
+- apiGroups: [""]
+  resources:
+  - secrets
+  verbs:
+  - list
```

Comment:
just curious, what is this needed for?

Reply:
Multicluster support, I think.

Reply:
Correct - multicluster.

@howardjohn , if curious, refer to:

* https://github.com/kiali/helm-charts/pull/30
* https://github.com/kiali/kiali/issues/1833

Reply:
For the code (and comments describing it), see: https://github.com/kiali/kiali/blob/v1.29.0/business/mesh.go#L233-L251

Reply:
We are introducing experimental support around advanced deployment models/multi-cluster and part of this is to start trying to "discover" mesh topology.

Reply:
@howardjohn , would love it if you gave this stuff a whirl.

Reply:
Nice! I did give 1.29 a shot yesterday, coincidentally, but didn't try out multicluster. cc @stevenctl @nmittler if you are interested. I'll give it a shot soon

Reply:
> just curious, what is this needed for?

It's for reading the secrets created for remote cluster discover (as stated here: https://istio.io/latest/docs/setup/install/multicluster/multi-primary_multi-network/#enable-endpoint-discovery). This way, Kiali can also do it's own discovery.